packages/data-transport-layer/src/services/l1-ingestion/service.ts

```diff
@@ -201,10 +201,13 @@ export class L1IngestionService extends BaseService<L1IngestionServiceOptions> {
         }
       } catch (err) {
         if (!this.running || this.options.dangerouslyCatchAllErrors) {
-          this.logger.error('Caught an unhandled error', { err })
+          this.logger.error('Caught an unhandled error', {
+            message: err.toString(),
+            stack: err.stack,
+            code: err.code,
+          })
```

Comment:
This seems like it's masking the underlying issue; why doesn't `{ err }` work?

Reply:
This fixes the underlying issue because using just `{ err }` results in logging an empty object

Reply:
I needed this fix to observe where problems were happening. After this change, I'm able to see the exact line of the problem where before it would be a mysterious empty object